config/jobs/sync-ghrepos-secrets-vars/sync-ghrepos-secrets-vars.yaml

```diff
@@ -0,0 +1,26 @@
+postsubmits:
+  falcosecurity/test-infra:
+    - name: sync-ghrepos-secrets-vars-post-submit
+      branches:
+        - ^master$
+      decorate: true
+      max_concurrency: 1
+      skip_report: false
+      run_if_changed: '^config/pigeon.yaml$|^config/jobs/pigeon/.*'
```

Comment:
I think we must have 2 different postsubmits jobs:
* one to run pigeon
* one to rebuild and push the image

And a presubmit job to test build the image.

Reply:
Good point! Will update it